engine/graphics/src/graphics.cpp

```diff
@@ -129,6 +130,7 @@ namespace dmGraphics
             GRAPHICS_ENUM_TO_STR_CASE(TEXTURE_TYPE_2D);
             GRAPHICS_ENUM_TO_STR_CASE(TEXTURE_TYPE_2D_ARRAY);
             GRAPHICS_ENUM_TO_STR_CASE(TEXTURE_TYPE_CUBE_MAP);
+            GRAPHICS_ENUM_TO_STR_CASE(TEXTURE_TYPE_IMAGE_2D);
```

Comment:
The "image 2d" is a new texture type in the engine now, compute shaders can't write to regular textures so we need a "storage image" to write to.

Reply:
@JCash we haven't exposed support for it, but yes it is possible. We should add support for creating all types of textures TBH